Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -93,7 +93,10 @@ import BraintreeDataCollector
     @objc(initWithAPIClient:universalLink:)
     public convenience init(apiClient: BTAPIClient, universalLink: URL) {
         self.init(apiClient: apiClient)
-        self.universalLink = universalLink
+        
+        /// appending a PayPal app switch specific path to verify we are in the correct flow when
+        /// `canHandleReturnURL` is called
+        self.universalLink = universalLink.appendingPathComponent("braintreeAppSwitchPayPal")
```

Comment:
Maybe you thought through this idea already - but we could instead just tell merchants to set different paths for each Payment method return vs having us manipulating their path.

Ex: `uber.com/venmo-payment-returns` and `uber.com/paypal-payment-returns`

Thoughts?

Reply:
That would still run into the same issue. If a merchant sent us `https://uber.com/paypal-payments-returns/success` that would pass our current checks of `https` and path contains `success`: https://github.com/braintree/braintree_ios/blob/57bf620ffacd0829bf368f11bf8ec6261c5ab595/Sources/BraintreeVenmo/BTVenmoAppSwitchReturnURL.swift#L71

This would return true and attempt to call `handleReturnURL` in the wrong feature client under this set up.

Reply:
Interesting! So we never actually compare the returnURL against the URL that the merchant originally passed into the `BTVenmoClient` or `BTPayPalClient`

Reply:
That's correct, I think merchants _should_ be able to use 1 URL for both if they want. This is also what we do today for our own demo.